02-peer-protocol.md

```diff
@@ -233,6 +233,8 @@ The sending node:
     - MAY include `upfront_shutdown_script`.
   - if it includes `open_channel_tlvs`:
     - MUST include `upfront_shutdown_script`.
+  - if `option_channel_type` is negotiated:
```

Comment:
I think eclair does this in the wild right? As in it'll always send a `channel_type` even if one wasn't sent over in the initial handshake?

Reply:
Exactly, we always send back the `channel_type` that is used (because when sending `accept_channel` we now know what `channel_type` is actually used). It's just about sending explicitly something that would otherwise be implicit.

Reply:
Tacked on